Client/Frontend/Browser/ReaderModeBarView.swift

```diff
@@ -8,6 +8,32 @@ import SnapKit
 
 enum ReaderModeBarButtonType {
     case MarkAsRead, MarkAsUnread, Settings, AddToReadingList, RemoveFromReadingList
+
+    private var description: String {
```

Comment:
Yep, I will (and will also look if I there is  some Swift protocol to adopt to formally mark the localizedDescription property as available). As this will require changes in more than one place etc., I will do it later today to be able to compile/test.